src/index.ts

```diff
@@ -537,10 +539,12 @@ export class Bigtable {
     this.projectName = `projects/${this.projectId}`;
     this.shouldReplaceProjectIdToken = this.projectId === '{{projectId}}';
 
-    if (options.metricsEnabled === false) {
-      this.metricsEnabled = false;
+    if (options.metricsEnabled === true) {
+      this._metricsConfigManager = new ClientSideMetricsConfigManager([
+        new GCPMetricsHandler(options as ClientOptions),
+      ]);
     } else {
-      this.metricsEnabled = true;
+      this._metricsConfigManager = new ClientSideMetricsConfigManager([]);
```

Comment:
not: looking at this again, maybe we should structure this like this?

```
if (options.metricsEnabled === true) {
     metric_handlers = [new GCPMetricsHandler(options as ClientOptions)]
} else {
    metric_handlers = []
}
this._metricsConfigManager = new ClientSideMetricsConfigManager(metric_handlers);
```

Reply:
Yup. Done. Simple change.